tests/integration-tests/conftest.py

```diff
@@ -42,12 +44,14 @@ def pytest_addoption(parser):
     parser.addoption("--oss", help="OSs under test", default=["alinux"], nargs="+")
     parser.addoption("--schedulers", help="schedulers under test", default=["slurm"], nargs="+")
     parser.addoption("--tests-log-file", help="file used to write test logs", default="pytest.log")
-    parser.addoption("--custom-node-url", help="url to a custom node package")
-    parser.addoption("--custom-cookbook-url", help="url to a custom cookbook package")
-    parser.addoption("--custom-template-url", help="url to a custom cfn template")
     parser.addoption("--output-dir", help="output dir for tests artifacts")
     parser.addoption("--key-name", help="key to use for EC2 instances", type=str, required=True)
     parser.addoption("--key-path", help="key path to use for SSH connections", type=str)
+    parser.addoption("--custom-chef-cookbook", help="url to a custom cookbook package")
+    parser.addoption("--custom-awsbatch-template-url", help="url to a custom awsbatch template")
+    parser.addoption("--template-url", help="url to a custom cfn template")
```

Comment:
I'd remove the `url `suffix and add the `custom` prefix to be aligned with other parameters

Reply:
The idea was to align the pytest options with the parameters we use in the pcluster config file while having the test_runner options defined with more aligned and explicit names.